easybuild/easyblocks/g/gromacs.py

```diff
@@ -475,13 +536,22 @@ def sanity_check_step(self):
         lib_files = []
         bin_files = []
 
-        for configopts in configopts_list:
-            # add the _d suffix to the suffix, in case of the double precission
-            if re.search('DGMX_DOUBLE=(ON|YES|TRUE|Y|[1-9])', configopts, re.I):
-                suff = '_d'
+        dsuff = None
+        if not get_software_root('CUDA'):
+            for configopts in configopts_list:
+                # add the _d suffix to the suffix, in case of double precission
+                if self.DP_pattern in configopts:
+                    dsuff = '_d'
+
+        if dsuff:
+            suffixes.extend([dsuff])
 
-            lib_files.extend(['lib%s%s.%s' % (libname, suff, self.libext) for libname in libnames])
-            bin_files.extend([b + suff for b in bins])
+        lib_files.extend([
+            'lib%s%s.%s' % (libname, suff, self.libext) for libname in libnames + mpi_libnames for suff in suffixes
+        ])
```

Comment:
I would replace `libname` with `x` here, which makes it shorter and more readable (due to the `libname` vs `libnames`):

```python
lib_files.extend(['lib%s%s.%s' % (x, suff, self.libext) for x in libnames + mpi_libnames for suff in suffixes])

Reply:
Done